server/loopbackconnectorsqlite3.js

```diff
@@ -0,0 +1,43 @@
+var sqlite3 = require('loopback-connector-sqlite3').verbose();
+
+
+function dosqlite3(result, callback) {
+  var db = new sqlite3.Database(':memory:');
```

Comment:
this was added accidentally. Not supposed to be here. I only have a test for raw sqlite, not for the connector.